templates/recipes/recipes_detail.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block css %}
+<link rel="stylesheet" href="{% static 'css/recipes/recipes-detail.css' %}">
+{% endblock css %}
+
+{% block content %}
+<div class="jumbotron">
+    <h1 class="display-4">{{ object.name }}</h1>
+    <p class="lead">Cook Time: {{ object.cook_time_in_minutes }} minutes</p>
+    <p>Makes {{ object.portion_number }} servings</p>
+    <hr class="my-4">
+
+    <div id='image-ingredients-container'>
+        <img id="detail-photo" src="{{ object.photo.url }}">
+        <div class="card border-primary" id='ingredients-card'>
+            <div class="card-header">ALL INGREDIENTS</div>
+            <div class="card-body">
+                <p class='sketchy-font' id='ingredients'>
+                    {% for ingredient in ingredient_list %}
+                        {% if ingredient.measurement.measurement == 'to taste' %}
+                            <!-- do nothing -->
+                        {% elif ingredient.measurement.measurement == 'Splash' %}
+                            A splash of {{ ingredient.ingredient }}<br>
+                        {% elif ingredient.measurement.measurement == 'Whole' and ingredient.quantity > 1 %}
+                            {{ingredient.quantity|floatformat}} {{ ingredient.ingredient }}s<br>
+                        {% elif ingredient.measurement.measurement == 'Whole' %}
+                            {{ingredient.quantity|floatformat}} {{ ingredient.ingredient }}<br>
+                        {% elif ingredient.quantity > 1 %}
+                            {{ingredient.quantity|floatformat}} {{ingredient.measurement}}s {{ ingredient.ingredient }}<br>
+                        {% else %}
+                            {{ingredient.quantity|floatformat}} {{ingredient.measurement}} {{ ingredient.ingredient }}<br>
+                        {% endif %}
+                    {% endfor %}
+                    {% for ingredient in ingredient_list %}
+                        {% if ingredient.measurement.measurement == 'to taste' %}
+                            {{ ingredient.ingredient }} to taste<br>
+                        {% else %}
+                            <!-- do nothing -->
+                        {% endif %}
```

Comment:
This for loop seems redundant with the `L21-22`. Is it because you wanted to add to-taste ingredients in the end? If so, you shouldn't need `L22`.